include/blas/device_blas.hh

```diff
@@ -44,6 +44,66 @@ void axpy(
     std::complex<double> *dy, int64_t incdy,
     blas::Queue& queue);
 
+// -----------------------------------------------------------------------------
+// dot
+void dot(
+    int64_t n,
+    float *dx, int64_t incdx,
+    float *dy, int64_t incdy,
```

Comment:
These should be `float const* dx` and `float const* dy`. Similar for double & complex. Compare src/dot.cc. That should be true in all these files.

Reply:
Could actually be `x` and `incx`, `y` and `incy`. I've been dropping the `d` to make diff comparisons easier.

Reply:
This will need to be updated then for the other device level 1 BLAS routines I've added. I am fine removing the "d" aspect to keep diffs cleaner

Reply:
I updated `axpy` and `nrm2` in another PR.